x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/search/action/GetAsyncSearchAction.java

```diff
@@ -63,12 +60,7 @@ public void writeTo(StreamOutput out) throws IOException {
 
         @Override
         public ActionRequestValidationException validate() {
-            ActionRequestValidationException validationException = null;
-            if (keepAlive.getMillis() != -1 && keepAlive.getMillis() < MIN_KEEP_ALIVE) {
-                validationException =
-                    addValidationError("keep_alive must be greater than 1 minute, got:" + keepAlive.toString(), validationException);
-            }
-            return validationException;
```

Comment:
I am curious, what triggered this change?

Reply:
The fact that we use very small value of keep_alive in tests to check the automatic cancellation.

Reply:
I see, while less than one minute does not make sense for submit, it does for get, probably.